giraffa-core/build.gradle

```diff
@@ -47,6 +50,12 @@ project(':giraffa-core') {
     }
   }
 
+  sourceSets {
+    generated{
+      java.srcDir "${buildDir}/generated/java/"
+    }
+  }
+
```

Comment:
Seems we need to add more here or remove this block.

If you add:
```
  compileGeneratedJava {
    classpath = configurations.compile
  }

  compileJava {
    dependsOn(compileGeneratedJava)
    source    += sourceSets.generated.java
  }
```
You can bring the error count down to just 15 compilation failures rather than the hundreds. The 15 failures are all around the *_jsp.java generated files.